.github/workflows/apim-apk-agent-pr-build.yml

```
@@ -0,0 +1,52 @@
+  contents: read
+
+concurrency:
+  group: apim-apk-agent-pr-${{ github.event.number }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v3
```

actionlint (1.7.12)

[error] the runner of "actions/setup-go@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)